devops/test_configs.json

```diff
@@ -1,8 +1,8 @@
 {
   "lts": [
     {
-      "config": "l0_ocl_gen9",
-      "name": "L0/OCL GEN9 LLVM Test Suite",
+      "config": "lin_gen9",
+      "name": "SYCL End-to-End tests on Intel GEN9 GPU on Linux",
```

Comment:
I suggest we keep this name short, so it can be displayed in PR.
<img width="670" alt="image" src="https://github.com/intel/llvm/assets/411965/c4ebe97d-ecab-4a38-a1cf-628f368885ae">